lib/marked.js

```diff
@@ -836,9 +839,10 @@ Renderer.prototype.hr = function() {
   return this.options.xhtml ? '<hr/>\n' : '<hr>\n';
 };
 
-Renderer.prototype.list = function(body, ordered) {
-  var type = ordered ? 'ol' : 'ul';
-  return '<' + type + '>\n' + body + '</' + type + '>\n';
+Renderer.prototype.list = function(body, ordered, start) {
+  var type = ordered ? 'ol' : 'ul',
+      startatt = (ordered && +start !== 1) ? (' start="' + start + '"') : '';
```

Comment:
Why are you casting `+start` here? Isn't it already a number from `+bull` above?